### DISCUSSION TITLE
Parallel computing

### DISCUSSION THREAD

**Discussion by kasia129** [capytaine/capytaine#127] [Q&A] [answered]
I have noticed that on some machines Capytaine runs on multiple cores in parallel, while on others it only uses a single core. How is the possibility of parallel computations implemented din Capytaine, and what are the prerequisites/dependencies to be able to use this functionality? I'm running it on Windows within a bigger python code.

**Comment by mancellin** [comment, marked as answer]
The current parallelization of the code relies of OpenMP in two places: in the Fortran implementation of the Green function bundled with Capytaine and in Numpy's linear algebra routines. It is normally controlled by the environment variables controlling OpenMP (see #47 and #65).

A similar issue of uncontrollable parallelization has been mentioned in #65, there hasn't been anything new in Capytaine since then.

I have some plans (#84) to provides more options for parallelization, but it is not done yet.
